renovate.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":dependencyDashboard",
```

Comment:
This is a new generated issue that can be used to easily visualize the packages in use as well as manage any PRs